force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -299,16 +359,10 @@ public with sharing class FlowControlLoopsCollections {
      * @return The Map of fruits and their quantities.
      */
     public static Map<String, Integer> addItemsToMap() {
-        // Initialize an empty Map
-
-        // Add key-value pairs to the Map
-        // add Apples => 5
-        // add Oranges => 10
-        // add Bananas => 15
-        // add Pears => 20
-        // add Grapes => 25
-
-        return null; // Replace null with the variable you used to store the result
+        Map<String, Integer> fruits = new Map<String, Integer>{'Apples' => 5, 'Oranges' => 10, 'Bananas' => 15, 'Pears' => 20};
+        // values can be added when the map is created or using put
+        fruits.put('Grapes',25);
+        return fruits; 
```

Comment:
Nice work using both ways to build a Map! I always use the method on line 362 if I know the values as I am writing.